crates/sui-storage/src/http_key_value_store.rs

```diff
@@ -86,6 +86,23 @@ pub enum Key {
     ObjectKey(ObjectID, VersionNumber),
 }
 
+impl Key {
+    /// Return a string representation of the key type
+    pub fn ty(&self) -> &'static str {
+        match self {
+            Key::Tx(_) => "tx",
+            Key::Fx(_) => "fx",
+            Key::Events(_) => "ev",
+            Key::CheckpointContents(_) => "cc",
+            Key::CheckpointSummary(_) => "cs",
+            Key::CheckpointContentsByDigest(_) => "cc",
+            Key::CheckpointSummaryByDigest(_) => "cs",
+            Key::TxToCheckpoint(_) => "tx2c",
+            Key::ObjectKey(_, _) => "ob",
+        }
```

Comment:
this could also be implemented as `key_to_path_elements().1` to avoid duplication. feel free to make `key_to_path_element` into a member function, i'm not sure why it isn't already